packages/flutter/lib/src/widgets/form.dart

```diff
@@ -209,6 +209,15 @@ class FormState extends State<Form> {
       hasError = !field.validate() || hasError;
     return !hasError;
   }
+
+  /// Check whether all the fields in this form are valid without changing
+  /// the display state.
```

Comment:
Add

```
See also:

  * [validate], which will validate and show any error on screen.
```

And also add a similar see also section to validate.

Reply:
Done.